styles.css

```diff
@@ -0,0 +1,1985 @@
+    font-family: 'BenchNine', sans-serif;
+    font-weight: 700;
+    max-width: 375px;
+}
+
+.button-reset {
+    background: #6bf99c;
+    border: 6px;
+    cursor: pointer;
+    display: block;
+    font-family: 'Teko', sans-serif;
+    font-size: 32px;
+    font-weight: 700;
+    left: 35px;
+    right: 61px;
+    height: 87px;
+    margin-top: 45px;
+    padding-bottom: 10px;
+    padding-left: 21px;
+    padding-right: 20px;
+    padding-top: 15px;
+    position: relative;
+    text-align: center;
+}
+
+​ .button-reset[disabled] {
+    cursor: default;
+}
+
+.button-reset:hover {
+    background: #fff;
+    border: 6px solid #6bf99c;
+    color: #000;
+    padding-bottom: 1px;
+    padding-left: 12px;
+    padding-right: 11px;
+    padding-top: 9px;
+
+}
+
+input,
+select,
+textarea {
+    background-color: #fff;
+    font-family: inherit;
+    font-size: inherit;
+}
+
+select {
+    cursor: pointer;
+}
+
+textarea {
+    /*height: 50px;*/
+    /*width: 275px;*/
+
+}
+
+fieldset {
+    border: 0;
+    padding: 0px;
+    /*vertical-align: top;*/
+}
+
+fieldset label {
+    bottom: 8px;
+    position: relative;
+    bottom: 50px;
+    left: 37px;
+}
+
+fieldset span {
+    display: block;
+}
+
+fieldset+fieldset {
+    margin-top: 34px;
+}
+
+input[type="radio"],
+input[type="checkbox"] {
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    appearance: none;
+    background-color: #1e2019;
+    border: 3px solid #d1d5de;
+    height: 24px;
+    width: 24px;
+}
+
+input[type="radio"]:focus,
+input[type="checkbox"]:focus {
+    border: 3px solid #6bf99c;
+}
+
+input[type="radio"]:hover,
+input[type="checkbox"]:hover {
+    background-color: #1E2019;
+    border: 3px solid #6bf99c;
+}
+
+
+input[type="radio"] {
+    border-radius: 50%;
+}
+
+input[type="radio"]:checked+label,
+input[type="checkbox"]:checked+label {
+    border-bottom: 4px solid #6bf99c;
+    outline: none;
+}
+
+input[type="radio"]:checked,
+input[type="checkbox"]:checked {
+    background-color: #6bf99c;
+}
+
+input[type="text"]:required:valid,
+input[type="text"]:required:valid,
+input[type="email"]:required:valid {
+    background-color: #6bf99c;
+}
+
+input[type="text"]:default,
+input[type="email"]:default,
+input[type="text"]:default {
+    background-color: #d1d5de;
+}
+
+input[type="text"]:focus,
+input[type="email"]:focus,
+input[type="text"]:focus {
+    background-color: #fff;
+}
+
+/*
+input[type="text"]:invalid,
+input[type="email"]:invalid,
+input[type="text"]:invalid {
+    background-color: #C61A3A;
+}
+*/
+
+
+input[type="text"]:invalid {}
+
+input[type="text"]:placeholder-shown {}
+
+input[type="text"]:not(placeholder-shown) {}
+
+input[type="text"]:not (:placeholder-shown):invalid {
+    background-color: orange;
+}
+
+textarea:focus {
+    border: 3px solid #6bf99c;
+    outline: none;
+}
+
+textarea:default {
+    background-color: #d1d5de;
+}
+
+div {
+    background-color: #1e2019;
+    color: #eaecf0;
+    min-width: auto;
```

Comment:
`min-width` is only meaningful if the `width` is set in percent or some other flexible unit. Since here your `width` is set as 327px, so `min-width` won't change that.